tasks/javascript.js

```diff
@@ -113,6 +115,17 @@ gulp.task('javascript:lint', function () {
 
 // karma unit tests in local library only
 gulp.task('javascript:unit', function () {
+  var reporters = cliArgs.karmareporter;
+  if (reporters.constructor === Array) {
+    reporters = Array;
```

Comment:
There are several ways to check if a variable is an `Array`, this being one of them. 
Think of it as `typeof myvar === 'array'` (which doesn't work because `typeof anyArray` returns `'object'`)